src/gltf-component-mappings.js

```diff
@@ -181,3 +190,74 @@ AFRAME.GLTFModelPlus.registerComponent("spawner", "spawner", (el, componentName,
   });
   el.setAttribute("hoverable", "");
 });
+
+const publicComponents = {
```

Comment:
Nice, I think this is much better than adding an additional concept to all inflators.